src/components/Video/Video.tsx

```diff
@@ -123,14 +67,18 @@ const Video: React.FC<Props> = ({
         <div className={styles.info}>
           <h2 className={styles.title}>{title}</h2>
           <div className={styles.metaContainer}>
-            <div className={styles.meta}>{metaString}</div>
-            {isSeries && <div className={styles.seriesMeta}>{seriesMeta}</div>}
+            <div className={styles.meta}>{videoMeta}</div>
+            {isSeries && (
+              <div className={styles.seriesMeta}>
+                <strong>{seriesMeta}</strong> - {episodeTitle}
+              </div>
+            )}
           </div>
-          <CollapsibleText text={item.description} className={styles.description} maxHeight={isMobile ? 60 : 'none'} />
+          <CollapsibleText text={description} className={styles.description} maxHeight={isMobile ? 60 : 'none'} />
 
           <div className={styles.buttonBar}>
             {startWatchingButton}
-            {trailerItem && (
+            {hasTrailer && (
```

Comment:
Suggest we pull the trailer stuff out as a button as well

Reply:
Done!